zcash_client_backend/src/data_api/chain.rs

```diff
@@ -86,10 +86,10 @@
 
 use std::convert::Infallible;
 
+use incrementalmerkletree::frontier::CommitmentTree;
 use zcash_primitives::{
     block::BlockHash,
     consensus::{self, BlockHeight},
-    merkle_tree::CommitmentTree,
     sapling::{note_encryption::PreparedIncomingViewingKey, Nullifier},
```

Comment:
```suggestion
    sapling::{note_encryption::PreparedIncomingViewingKey, CommitmentTree, Nullifier},
```
(or change below to `sapling::CommitmentTree`).